### ISSUE TITLE
Difficult to remove support in some cases

### ISSUE THREAD

**Issue by kefir-** [alexrj/Slic3r#2061]
Latest slic3r seems to be working very well in my work so far, really nice! But I've come across a part where support is difficult to remove, and it's the filament drive for MakerGear M2. The part is this one:

View on github: https://github.com/MakerGear/M2/blob/master/Printed%20Parts/STL/M2%20Filament%20Drive%201.75mm.stl
Download from github: https://github.com/MakerGear/M2/raw/master/Printed%20Parts/STL/M2%20Filament%20Drive%201.75mm.stl
My gcode after slicing: http://kefirshare.dreamhosters.com/M2%20Filament%20Drive%201.75mm_20140528-004435_1.1.4-dev.gcode

When I slice this with support, the support material completely engulfs one section of the filament drive. It seems like a pretty difficult corner case, but I'm mentioning it anyway. Here's a few descriptive pictures:

This is one of the first layers, support is shown in blue.

![screenshot from 2014-05-28 01 11 03](https://cloud.githubusercontent.com/assets/684832/3099025/010db788-e5f5-11e3-885c-330b4704c40f.png)

Then, on top of the support, the part is created. The current layer is shown in blue, and this time it's the part that's being printed, not support. Observe the right side of the big circle:

![screenshot from 2014-05-28 01 11 18](https://cloud.githubusercontent.com/assets/684832/3099030/3725c342-e5f5-11e3-9221-d083264033d8.png)

A few layers later on, some more support is added. Note how this covers over the right side of the big circle completely, thereby enclosing it completely.

![screenshot from 2014-05-28 01 11 51](https://cloud.githubusercontent.com/assets/684832/3099036/4ff3ab0a-e5f5-11e3-8167-c670371c83e6.png)

This support is going to be difficult to remove, because the support must be broken apart inside the printed object before it can be taken out.

**Comment by lordofhyphens** [comment]
Agreed, small  holes make for painful support generation. I think bridge/overhang detection is still "broken"; will take some experimenting.
